### PR TITLE
changed AS15435 from safe to unsage

### PR DESCRIPTION
**Pull request by AlexStorm1313**
Update AS15435 from safe to unsafe. Potential proof: #745

### CODE REVIEW

**Comment by digizeph** [comment]
Thanks for the PR.

Here are some additional evidence:
* Lacks of ROV: https://stats.labs.apnic.net/rpki/AS15435
* Still signing all prefixes: https://radar.cloudflare.com/routing/as15435?dateRange=7d

image
image

File: data/operators.csv [hunk 1]
```diff
@@ -299,7 +299,7 @@ Istanbuldc Veri Merkezi,cloud,,unsafe,197328,6633
 Sprint Personal Communications Systems,transit,,unsafe,10507,6634
 Aura Fiber,ISP,,safe,204274,6735
 Kaisanet Oy,ISP,,unsafe,13170,6773
-DELTA Fiber,ISP,signed + filtering,safe,15435,6903
+DELTA Fiber,ISP,unsafe,15435,6903
```
Comment (digizeph) [inline data/operators.csv:302]: Add `signed` column:

```suggestion
DELTA Fiber,ISP,signed,unsafe,15435,6903
```

**Comment by AlexStorm1313** [comment]
@digizeph Is my understanding correct that the prefixes are signed, but not checked by the ISP and is there a way to validate this yourself (locally)?

**Comment by digizeph** [comment]
> @digizeph Is my understanding correct that the prefixes are signed, but not checked by the ISP and is there a way to validate this yourself (locally)?

There are two parts for routing security with RPKI for an ISP:
1. protect its own IP prefixes by signing its prefixes, so that BGP hijackers who attempted to announce its prefixes will be rejected by other networks who filters RPKI invalid announces.
2. protect all other networks by dropping RPKI invalid announcesments received on its routers (i.e. route origin validation (ROV)).

AS15435 used to do both, i.e. sign its routes and filter RPKI invalid announcements. Now it stopped filtering invalid routes and only signs its prefixes.

For checking prefix signing status for any ASN, you can use Cloudflare Radar routing stats page: https://stats.labs.apnic.net/rpki/AS15435

For checking RPKI invalid filtering, the check you did you do on isbgpsafeyet.com is the simplest way to do so. You can also check other public measurements to see results from different vantage points: https://radar.cloudflare.com/routing/as15435?dateRange=7d.

Hope this helps.

**Review by digizeph — APPROVED** [review]

**Pull request merged**